package/CHANGELOG

```diff
@@ -13,13 +13,14 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/?? IAlibay, melomcr
+??/??/?? IAlibay, melomcr, rsexton2
 
  * 2.1.0
 
 Fixes
 
 Enhancements
+  * Added frames processing to AnalysisBase.run (Issue #1985)
```

Comment:
Can we use something similar to the versionchanged entry here? Whilst I'm sure it's not something most people read, it is where we direct folks to read about new changes, so being a bit more descriptive doesn't hurt.

Reply:
Added more explicit description